sql-plugin/src/main/spark311/scala/org/apache/spark/sql/rapids/shims/GpuAscii.scala

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+
```

Comment:
Nit: extraneous whitespace-only change to this file, inconsistent with spacing in other files.